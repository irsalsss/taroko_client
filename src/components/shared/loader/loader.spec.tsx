import { render } from "@/utils/test/react-testing-setup";
import Loader from "./loader";

describe("Loader", () => {
  it("should render loader successfully", () => {
    const { baseElement } = render(<Loader />);

    expect(baseElement).toMatchSnapshot();
    expect(baseElement).toBeTruthy();
  });
});
