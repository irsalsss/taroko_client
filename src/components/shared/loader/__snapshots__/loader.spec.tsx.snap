// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should render loader successfully 1`] = `
<body>
  <div>
    <span
      class="loader"
    />
  </div>
</body>
`;
